x/mongo/driver/auth/creds/aws_credential_providers/static_provider.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) MongoDB, Inc. 2023-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package awscredproviders
+
+import (
+	"errors"
+
+	"go.mongodb.org/mongo-driver/x/mongo/driver/auth/internal/aws/credentials"
+)
+
+// StaticProviderName provides a name of Static provider
+const StaticProviderName = "StaticProvider"
```

Comment:
This constant has no usages outside of the `awscredproviders` package, so does not need to be exported.